cockroachcloud/quickstart.md

```diff
@@ -380,89 +93,40 @@ For reference information about connecting to CockroachDB with supported client
     git clone https://github.com/cockroachdb/quickstart-code-samples
     ~~~
 
-1. Navigate to the `go` directory of the repo:
+1. Navigate to the `node` directory of the repo:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    cd quickstart-code-samples/go
+    cd quickstart-code-samples/node
     ~~~
 
     The code sample in this directory does the following:
-      1. Connects to {{ site.data.products.db }} with the [pgx driver](https://github.com/jackc/pgx) using the connection string set in the `DATABASE_URL` environment variable.
+      1. Connects to {{ site.data.products.db }} with the [node-postgres driver](https://node-postgres.com) using the connection string set in the `DATABASE_URL` environment variable.
       1. Creates a table.
       1. Inserts some data into the table.
       1. Reads the inserted data.
       1. Prints the data to the terminal.
 
-1. Initialize and run the app:
+1. Install the code dependencies:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ go mod init basic-sample
+    npm install
     ~~~
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    go mod tidy
-    ~~~
+1. Run the code:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ go run main.go
+    node app.js
     ~~~
 
     The output will look like this:
 
     ~~~
     Hello world!
     ~~~
-
-</section>
-{% comment %} End go {% endcomment %}
-
-<section class="filter-content" markdown="1" data-scope="java">
-
-1. Clone the `quickstart-code-samples` repo:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    git clone https://github.com/cockroachdb/quickstart-code-samples
-    ~~~
-
-1. Navigate to the `java` directory of the repo:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    cd quickstart-code-samples/java
-    ~~~
-
-    The code sample in this directory does the following:
-      1. Connects to {{ site.data.products.db }} with the [JDBC driver](https://jdbc.postgresql.org) using the JDBC connection string set in the `JDBC_DATABASE_URL` environment variable.
-      1. Creates a table.
-      1. Inserts some data into the table.
-      1. Reads the inserted data.
-      1. Prints the data to the terminal.
-
-1. Run the application using `gradlew`:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    ./gradlew run
-    ~~~
-
-    The output should look like this:
-
-    ~~~
-    > Task :app:run
-    Hello world!
-
-    BUILD SUCCESSFUL in 3s
-    2 actionable tasks: 2 executed
-    ~~~
-
-</section>
-{% comment %} End java {% endcomment %}
-
+    
 ## Next steps
```

Comment:
I think we can improve these last two sections to be more helpful/concise now that the rest of the Quickstart is also streamlined.

Suggestion: combine the two sections into one (maybe learn more?) and consolidate/prune the two lists. I imagine something like the following (but better worded):

```
## Learn more

Now that you have a free Serverless cluster running, try out the following:

- Build a simple CRUD app
- Create and manage SQL users
- Migrate your existing data

This page highlights just one way you can get started with CockroachDB. For information on other options that are available when creating a CockroachDB cluster, see the following:

- Create a dedicated cluster
    - [More links maybe]
- Create SH cluster
     - [more links maybe]
```